core/src/commonMain/kotlin/com/sourcepoint/mobile_core/storage/PlatformStorage.kt

```diff
@@ -0,0 +1,14 @@
+package com.sourcepoint.mobile_core.storage
+
+import com.russhwolf.settings.MapSettings
+import com.russhwolf.settings.Settings
+
+expect fun getStorage(): Settings
+
+fun getStorageOrDefault() = try {
+    getStorage()
+} catch (e: NotImplementedError) {
+    MapSettings()
+} catch (e: NullPointerException) {
+    MapSettings()
```

Comment:
@andresilveirah I'd rather catch the most generic `Exception`, but that's not a dealbreaker if you guarantee that's the only two types of errors possible

Reply:
I thought about it as well, but maybe we want to see it fail explicitly if some other exception happens.